torchtext/experimental/vectors.py

```diff
@@ -0,0 +1,188 @@
+import csv
+import logging
+import os
+
+import torch
+from torch import Tensor
+import torch.nn as nn
+from tqdm import tqdm
+
+from torchtext.utils import (
+    download_from_url,
+    extract_archive
+)
+
+logger = logging.getLogger(__name__)
+
+
+def _load_token_and_vectors_from_file(file_path):
+    stoi, tokens, vectors, dup_tokens = {}, [], [], []
+    with open(file_path, 'rb') as f:
+        for line in tqdm(f):
+            token, entries = line.rstrip().split(b" ", 1)
+            vector = torch.tensor([float(c) for c in entries.split(b" ")], dtype=torch.float)
+
+            try:
+                if isinstance(token, bytes):
+                    token = token.decode('utf-8')
+            except UnicodeDecodeError:
+                logger.info("Skipping non-UTF8 token {}".format(repr(token)))
+                continue
+
+            if token in stoi:
+                logger.info("Found duplicate entries for token {} on line {}.".format(repr(token), len(vectors) + 1))
+                dup_tokens.append(token)
+                continue
+
+            stoi[token] = len(vectors)
+            tokens.append(token)
+            vectors.append(vector)
+
+    return tokens, vectors, dup_tokens
+
+
+def fast_text(language="en", unk_tensor=None, root='.data'):
+    r"""Create a fast text Vectors object.
+
+    Args:
+        language (str): the language to use for FastText.
```

Comment:
We should be explicit for the language options supported.

Reply:
I believe there is actually a list of 157 languages supported. I will add the URL where they can find out what the options are (i.e. https://fasttext.cc/docs/en/language-identification.html)